### ISSUE TITLE
Incomplete `Extending Workbench` page

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode-docs#2119]
This page is very incomplete... maybe we should get @bpasero's input on it?

https://vscode-ext-docs.azurewebsites.net/api/extension-capabilities/extending-workbench

**Comment by bpasero** [comment]
Wasn't @isidorn supposed to cover that page?

**Comment by isidorn** [comment]
Yeah I covered it incompletely.
Basically I covered all the topics that @octref outlined
@joaomoreno @bpasero it would be great if you can list the topics that seem to be missing

**Comment by octref** [comment]
Yeah that's why I assigned @bpasero for testing https://github.com/Microsoft/vscode/issues/62106. 

@isidorn or @bpasero: Would you have time in debt week to update the doc, so it's as informative as the other `Extension Capabilities` topics such as [Overview](https://vscode-ext-docs.azurewebsites.net/api/extension-capabilities/overview) and [Theming](https://vscode-ext-docs.azurewebsites.net/api/extension-capabilities/theming)?

**Comment by isidorn** [comment]
@octref yes I would have time to update the doc in debt week. However the workbench is a very global topic and I am not sure what areas are covered in other documents. So it would be really helpful for me if we could figure out what this docuemnt should contain first.

**Comment by bpasero** [comment]
@octref sure, just let me know when I should review (e.g. is the topic complete?)
